doc/Requirements Specification/Parts/Non-Functional Requirements.lyx

```diff
@@ -858,7 +870,20 @@ To analyse the entire project, there is an entry
 \begin_inset Quotes eld
 \end_inset
 
-Analyse with Beagle
+Analyse with 
+\begin_inset ERT
+status open
+
+\begin_layout Plain Layout
+
+
+\backslash
+gls{Beagle}
+\end_layout
```

Comment:
I thought that it might make sense not to reference Beagle in this citations, as they give exact text to be printed and contain no semantic.